CMSgov/mint-app
# [EASI-3945] Collaborator Added notification BE

# EASI-3945
## Changes and Description

- New added as collaborator notification type
- Plan Collaborator Get by ID data loader
- Plan Collaborator general refactoring for naming consistency
- Plan Collaborator SQL files moved to sql queries package


## How to test this change
1. Bring up the application (The steps below can use either the postman collection or the FE)
2. Add yourself as a collaborator to a model plan. 
3. Verify that you got an email in mail catcher
4. Use Postman to view the new notification type

If desired, update your notification preferences to turn off email for added as collaborator.
5. Repeat the above steps, but confirm you don't get an email for being added as a collaborator
6. Create a new model plan, ensure you don't get notified that you were added as a collaborator.



## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
